CHANGELOG.md

```diff
@@ -24,16 +24,26 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/report/compare/release-3.11.4...master
+[Unreleased]: https://github.com/atlassian/report/compare/release-3.11.5...master
 
-### Fixed
-- Restore missing log label in failed regression test
+## [3.11.5] - 2022-10-20
+[3.11.5]: https://github.com/atlassian/report/compare/release-3.11.4...release-3.11.5
+
+### Changed
+- Restore missing label for failed report's assertion message in `RelativeNonparametricPerformanceJudge` [JPERF-832].
```

Comment:
So did we need that label for readability or for the internal overeager test?

Reply:
JPT test failed and thx to that I found out I removed it. And that contributes to log readability. So both.
After releasing this lib I'll modify JPT tests not to rely on report message

Reply:
> It does. It suggests that a feature (contract) was changed, ie. it's not compatible anymore. It's like Removed and Added at the same time. If no contract was changed, then it's just a transparent fix (Fixed).

I will change it to `### Fixed`